### DISCUSSION TITLE
Issue while saving patient question answers to EPIC sandbox using firely .net sdk

### DISCUSSION THREAD

**Discussion by fayazshaik07** [FirelyTeam/firely-net-sdk#2704] [Q&A]
Hi There

Our Patient facing app collects responses to some questions and based on the responses, provider would prescribe the medication.
In-order to save the responses temporarily (in the EPIC FHIR Sandbox), I'm using the below FHIR R4 API

https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/R4/Condition

My sample responses of the patient would look like below object:
`[
    {
        "name": "Acne Location",
        "value": "Neck",
    },
	{
	    "name": "Fever Check",
        "value": "No"
	}
    {
        "name": "Skin Infection Check",
        "value": "No"
    },
    {
        "name": "Outbreak - New/Old",
        "value": "No"
    },
    {
        "name": "Skin Rating",
        "value": "Oily"
    },
    {
        "name": "Acne Treatments",
        "value": "No"
    },
    {
        "name": "Liver or Kidney Problems",
        "value": "No"
    },
    {
        "name": "Any Medical Conditions",
        "value": "No"
    },
    {
        "name": "Side Effects or Allergies",
        "value": "No"
    },
    {
        "name": "Sex Assigned",
        "value": "Female"
    },
    {
        "name": "Pregnancy Check",
        "value": "No"
    }
]`

Could you please suggest the best resource from firely .net sdk for saving these responses against a patient to EPIC sandbox.
@mbaltus If possible, your help is appreciated. Thanks in Advance.

**Comment by mmsmits** [comment]
This is not a question related to the SDK I think.
If you need help with FHIR implementations, I would suggest to subscribe to http://chat.fhir.org/.
